src/jupytext_config/labconfig.py

```diff
@@ -12,10 +12,12 @@
 # import pprint
 from pathlib import Path
 
+DEFAULT_SETTINGS_FILE = (
+    Path.home() / ".jupyter" / "labconfig" / "default_setting_overrides.json"
```

Comment:
This can be changed by the user. I think a more robust solution would be directly get it from `jupyter_core`. Something like following:

```
from jupyter_core import paths

config_dir = paths.jupyter_config_dir()
settings_file = Path(config_dir) / "labconfig" / "default_setting_overrides.json"
```

Reply:
Thanks ! I'll update the PR accordingly